### ISSUE TITLE
Asking questions about your paper

### ISSUE THREAD

**Issue by pikalooo** [triplej0079/DeepIM#3]
Hi, your work on "Deep Graph Representation Learning and Optimization for Influence Maximization" is excellent and has been very inspiring to me. However, I have a few questions that I would like to ask you.
1.  I noticed that there is no Budget Constraint included in the code you have open-sourced. Could you please provide the relevant code for the Budget Constraint so that I can conduct comparative experiments?

2.   The setting of x_pred[x_pred>0.01] = 1 in genim.py is correct?  However, it's important to note that if using this threshold, x_hat will indeed be entirely set to 1, and the size of seed_num = int(x_hat.sum().item()) will become equal to the total number of nodes.

3.  In genim.py, is it necessary to set y = torch.where(y_hat > 0.05, 1, 0) during the inference process of the optimal seed set? This value is not involved in subsequent calculations in the code.

4. My designing loss functions is not very good. I didn't understand why the loss_inverse () function in genim.py differs from the formula (8) in the paper during the inference process.

5. In the inference process of genim.py, the loss obtained from loss_inverse() does not converge. Is this correct?

I am looking forward to your response, and I appreciate your assistance with this matter.

Best regards,
Zhang
